test/trento/users_test.exs

```diff
@@ -5,6 +5,49 @@ defmodule Trento.UsersTest do
   alias Trento.Users.User
   import Trento.Factory
 
+  describe "user profile" do
+    test "update_user_profile does not update the profile when fields are not present" do
```

Comment:
I'm not sure that the description of the test matches what the test does, am I missing something?

Reply:
It's for testing that the update is idempontent, no field passed, no field updated, this should match with the patch verb we want to implement